stage1/init/common/pod.go

```diff
@@ -564,8 +564,9 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		}
 	}
 
-	// Restrict access to sensitive paths (eg. procfs) and devices
-	if !insecureOptions.DisablePaths {
+	// Restrict access to sensitive paths (eg. procfs) and devices. For kvm flavor,
+	// all devices are stored inside the vm, without influence to the host.
```

Comment:
`protectSystemFiles` is also for other than devices paths, so this comment is a bit imprecise.
Let's change it to "for kvm flavor these paths are ..."